src/store/apiSlice.ts

```diff
@@ -311,3 +320,53 @@ export default class ApiSlice {
     this.redux.dispatch(apiSlice.actions.setAbortDelayedJob(abort_delayed_job));
   }
 }
+
+export const selectResetPassword = (store: ApiDataStore) => Object.values(store.logins)?.[0]?.attributes;
+
+type RequestResetPasswordConnection = {
```

Comment:
Connections should be in their own file in `src/connections`, not in the apiSlice file.